### ISSUE TITLE
NullPointerException caused by using BottomSheet#limit()

### ISSUE THREAD

**Issue by StefanDeBruijn** [soarcn/BottomSheet#55]
Description:
A NullPointerException is thrown when you set a menu resources with BottomSheet#sheet() that contains a small number (tested with 1~4) of items together with a limit, when clicking the item.

Code:
```java
sheetScripts = new BottomSheet.Builder(getActivity()).sheet(R.menu.menu).listener(new DialogInterface.OnClickListener() {
                    @Override
                    public void onClick(DialogInterface dialog, int which) {
                        dialog.dismiss();
                    }
                }).grid().limit(R.integer.bs_initial_grid_row).show();
```

The value of  ``` R.integer.bs_initial_grid_row ```  is 3

R.menu.menu:
```xml
<?xml version="1.0" encoding="utf-8"?>
<menu xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:id="@+id/some_id" android:title="some_title" android:icon="@mipmap/ic_launcher"/>
</menu>
```
Log:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.cocosw.bottomsheet.ActionMenu.removeInvisible()' on a null object reference
            at com.cocosw.bottomsheet.BottomSheet.updateSection(BottomSheet.java:457)
            at com.cocosw.bottomsheet.BottomSheet.showShortItems(BottomSheet.java:501)
            at com.cocosw.bottomsheet.BottomSheet.access$1800(BottomSheet.java:75)
            at com.cocosw.bottomsheet.BottomSheet$7.onDismiss(BottomSheet.java:559)
            at android.app.Dialog$ListenersHandler.handleMessage(Dialog.java:1263)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5254)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```

**Comment by soarcn** [comment]
Good, will fix this asap

**Comment by soarcn** [comment]
Fixing has been included in 1.1.1, available in maven repository soon.
